.github/workflows/main.yml

```
@@ -1,25 +1,35 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
 
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     container:
-      image: elementary/docker:unstable
+      image: elementary/docker:next-unstable
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
     - name: Install Dependencies
       run: |
         apt update
-        apt install -y exif meson
+        apt install -y exif librsvg2-bin meson
     - name: Test Metadata
       run: bash tests.sh
+
     - name: Build
       run: |
         meson build
-        ninja -C build
+        ninja -C build install
+
+    - name: Build (Early Access)
+      run: |
+        meson configure -Dearly_access=true
```

```suggestion
        meson configure build -Dearly_access=true
```

This does not work as expected. From meson's docs:

> You invoke `meson configure` by giving it the location of your build dir. If omitted, the current working directory is used instead.

Source: https://mesonbuild.com/Configuring-a-build-directory.html

---

Oops thanks for catching that